### ISSUE TITLE
[Docs] Notion M3~M5 Exit Criteria 신설 — GitHub 이슈 폐기 3 · 재정의 4 · 신설 6 정리

### ISSUE THREAD

**Issue by Feroninn** [grippers-intel/grippers#145]
@sysy009 @kica927

Notion 허브에 **M3 · M4 · M5 Exit Criteria 가 없어 GitHub 최신화가 막힌다**는 요청이 있어, §5-5 ~ §5-8 을 새로 작성했습니다.

https://app.notion.com/p/3bbe5823472881ea9910edbf8386ca43

이 이슈는 그 문서를 GitHub 에 반영하기 위한 작업 목록입니다. 발행·수정 권한이 Git Master 께 있어 정리만 해서 올립니다.

## 현황 — M3~M5 이슈 15건이 전부 구주제 기준입니다

8/13 주제 전환(장물 통과 → 장난감 정리) 이전에 발행되었고, **완료 조건이 비어 있거나 작업 내용을 그대로 복사한 상태**입니다. 지금 상태로는 무엇을 해야 닫히는지 판단할 수 없습니다.

구분 기준은 하나입니다 — **8/13 주제 전환 · 8/14 단안 전환 · 8/18 C270 ×2 로 전제가 바뀌었는가.**

## ❌ 폐기 3건

| # | 제목 | 사유 |
|---|---|---|
| #47 | IK 기반 자세 재조정 동작 검증 | **자세 재조정 보류**(PR #86). `φ` 계산 자체가 없어졌습니다 |
| #55 | 학습 기반 마진 결정 실험 | 마진은 개구부 통과 여유였는데 개구부가 없습니다. 파지 여유는 기하 폴백으로 충분 |
| #87 | Nuwa 드라이버 Pi 5 빌드 | 깊이 카메라 미사용 — **8/19 종료 완료** |

## 🔄 재정의 4건

| # | 현재 제목 | 바꿔야 할 것 |
|---|---|---|
| #48 | VLA 3파트 결합 End-to-end 추론 | **end-to-end VLA 를 명시적으로 포기**했습니다(허브 §1 피드백 대응표). → `language 모듈 결합 — 명령 해석이 FSM 을 관통` |
| #50 | VLA 결합 참여 (Vision측) | → `perception 모듈을 미션 파이프라인에 연결` *(제 담당, 이 방향으로 진행하겠습니다)* |
| #52 | 조명 조건별 성능 측정 (4×3 비교표) | 정상광/암실/옐로우룸 3조건이 사라졌습니다. → `학습 조건별 비교 — Synthetic only / +real fine-tune / Real only` |
| #121 | A1-b · depth cam 장착 위치 확정 | → `A1-b · 바닥 파지 리치 + 그리퍼 개구 폭 실측`. **미할당이며 현재 최장 경로입니다** |

## ➕ 신설 6건 — 이슈가 아예 없습니다

상위 4건은 **카메라를 한 대에서 두 대로 바꾸면서, 또는 깊이 센서를 빼면서 생긴 것**들입니다.

| 작업 | 왜 필요해졌나 | M | 제안 오너 |
|---|---|---|---|
| **두 시야 병합** — 경계 중복 검출 제거 + `track_id` 재부여 | C270 ×2 가 담당 경계를 공유합니다. 안 하면 **물체 하나가 둘로 보입니다** | M2 | 김동혁 |
| **박스 하단 ↔ 실제 접지선 편차 측정** | 운용 중에는 YOLO 박스 하단이 사람의 클릭을 대신합니다. **A2 는 이걸 재지 않습니다** | M2 | 김동혁 |
| **오차 예산 재작성** (`σ_depth` → 호모그래피) | 깊이 센서가 없어 오차 구성 자체가 바뀝니다. #106 과 연결 | M2 | 김동혁 |
| **검정 상자 ArUco 병행** | 검정은 LAB 에서 `L*` 만 남아 그림자와 구분 불가. 리스크 대응책이 작업으로 존재하지 않습니다 | M2 | 김희수 · 김동혁 |
| **동일 물체 반복 검출 방지** — 상자 영역 마스킹 + 처리 완료 목록 | 무한 루프 직결. 리스크표에 "도메인 테스트 필수"로 적혀 있으나 이슈가 없습니다 | M2 → M3 | 조현우 |
| **성공 기준 수치 확정** | §5-3 에서 "M2 실측 후"로 미뤄둔 항목. M4 측정 전에 숫자가 있어야 합니다 | M2 말 | 이승용 |

## 📅 기간 조정 제안

발표 9/8 과 포스터 제출 9/6 은 고정입니다. 뒤가 막힌 상태에서 M1 이 밀렸으므로, **§5 의 원칙대로 날짜가 아니라 범위를 깎는** 쪽으로 잡았습니다.

| 마일스톤 | 기존 | 조정 |
|---|---|---|
| M1 설계 | 8/13 – 8/18 | **8/13 – 8/20** (사실 인정) |
| M2 모듈 | 8/19 – 8/26 | **8/21 – 8/27** |
| M3 통합 | 8/27 – 8/31 | **8/28 – 9/1** |
| M4 MVP | 9/1 – 9/4 | **9/2 – 9/4** (3일) |
| M5 발표 | 9/5 – 9/8 | 유지 |

### 범위 축소 순서 (위에서부터 자른다)

1. 음성 입출력 → GUI 텍스트로 대체
2. FETCH 모드 → TIDY 단일로도 "상태 재관측 루프"는 온전히 시연됨
3. **파지 정책 학습(M2-A1)** → 기하 폴백 단독. 텔레오퍼레이션 데이터 수집 부담이 통째로 사라집니다
4. 대시보드 실시간성 → 정적 로그 뷰어

**거꾸로 건드리지 않을 것** — 상자 4개(무작위 기준선 25% 근거) · 2클래스 × 형상 3종(핵심 주장) · 실제 validation set(수치의 근거)

## 지금 막혀 있는 것

M1 잔여 6건 중 4건이 **#130 하나**에 걸려 있습니다. 측정 도구는 PR #129 로 머지되어 `tools/a2/` 에 있고, 남은 것은 카메라를 세우는 물리 작업뿐입니다.

#121 도 미할당이라 asset freeze → 프린트 → validation 연쇄를 막고 있습니다. **프린트 시간을 감안하면 이게 M2 전체를 좌우합니다.**

---

폐기·재정의는 제가 처리해도 되면 말씀 주세요. 신설 6건은 담당 배정이 필요해 Git Master 발행이 맞다고 봅니다.

**Comment by Feroninn** [comment]
@sysy009 @kica927

마일스톤 재배정 확인했습니다. **M3~M5 이슈 15건이 전부 마일스톤을 받았고, 폐기 3건(#47 · #55 · #87)과 재정의 4건도 반영되었습니다.** Notion 허브 §5-5 ~ §5-8 을 GitHub 현재 상태에 맞춰 최신화했습니다.

남은 어긋남 4건만 정리해 올립니다. 전부 **발행·선언 권한이 Git Master 께 있는 것**이라 제안만 드립니다.

---

## ① Milestone 기한이 구본에 그대로 있습니다

이슈 ↔ 마일스톤 **배정은 갱신**되었는데 **기한은 8/13 재조정본**입니다.

| 마일스톤 | GitHub 현재 기한 | Notion §5-9 (현행 기준) |
|---|---|---|
| M1 설계 | **8/18 (이미 지남)** | ~8/20 |
| M2 모듈 | 8/26 | **8/20 – 8/23** |
| M3 통합 | 8/31 | **8/26 – 9/1** |
| M4 MVP | 9/4 | 9/2 – 9/4 |
| M5 발표 | 9/8 | 9/5 – 9/8 |

M2 가 사흘 차이 납니다. 8/25 캡스톤 지원 때문에 M2 를 8/23 에 끊고 8/24~25 를 시연에 쓰기로 §5-9 에서 정한 건데, GitHub 만 보면 8/26 까지 여유가 있어 보입니다.

## ② 마일스톤이 없는 이슈 8건

가장 급한 **#146(8/25 시연)** 이 여기 들어 있습니다.

| # | 제목 | 제안 |
|---|---|---|
| #146 | 8/25 캡스톤 지원 시연 | **신규 마일스톤 "캡스톤 지원 시연 (8/25)"** — M2 도 M3 도 아닌 별도 목표라 §5-9 처럼 따로 세우는 게 맞다고 봅니다 |
| #155 | HEF 컴파일 호스트 확정 | M2 (M2-H1 이 여기 종속) |
| #156 | arm_driver_node 조용한 성공 처리 | M2 |
| #143 | find_box 가 opening_mm 에 None | M2 |
| #141 | spin_until_future_complete 실행기 분리 | M2 |
| #106 | docs 구주제 잔존 3건 | M2 |
| #85 | README 미반영 3건 | M4 (#54 README 최종화와 함께) |
| #145 | 이 이슈 | 상시 확인 |

## ③ 신설 6건은 아직 0건입니다

폐기·재정의는 다 반영되었고 M4-V1 은 #158 로 나왔는데, **신설 6건만 그대로 남아 있습니다.**

순서로는 **1번(두 시야 병합)이 가장 급합니다.** 내일 A2 가 끝나면 호모그래피가 2장 생기고, 그 즉시 경계 중복 검출이 실제 문제로 드러납니다. 제 담당 3건(1 · 2 · 3번)은 발행만 해 주시면 제가 받겠습니다.

## ④ #121 이 수치 없이 닫혔습니다

그리퍼 교체와 잡기 테스트는 끝났는데 **개구 폭 mm · 바닥 파지 리치 mm 두 값이 코멘트에 없습니다.** 제가 08-19 06:16 에 요청드렸으나 그 전에 닫혔습니다.

이 두 숫자가 STL 스케일의 **상한**이고, 하한은 카메라가 이미 정했습니다(50 mm · 최소 42 mm).

```
개구 폭 실측 → STL 스케일 확정 → asset freeze (#93) → 1차 프린트 (#103) → 실물 validation (#102)
```

**현재 8/25 시연의 최장 경로입니다.** 재개방하실지, 별도 이슈로 발행하실지 판단 부탁드립니다.

---

**Notion 최신화분** — §5 날짜표에 "이력이며 기준은 §5-9" 배너 · §5-4/§5-5 헤더 날짜 정정 · M4-V1 에 #158 연결 · 폐기 3건 종료 표기 · 재정의 4건 반영 표기 · 예산 집행액 74,600 → **147,600원** (잔액 152,400) · 카메라 수치를 확정 배치(1,650) 기준으로 통일

카메라 쪽은 2,050 가정 시절 수치(2.69 m · 15.7 px · 가림 124 mm)가 본문에 남아 있어 확정값(2.97 m · 14.3 px · 가림 377 mm)과 나란히 비교표로 바꿨습니다. 폐기값을 지우지 않은 건 "왜 1,650 인가"의 답이라서입니다.
